src/MonitorProfiler/MainProfiler/MainProfiler.cpp

```diff
@@ -91,3 +97,34 @@ HRESULT MainProfiler::InitializeLogging()
 
     return S_OK;
 }
+
+HRESULT MainProfiler::InitializeCommandServer()
+{
+    HRESULT hr = S_OK;
+
+    tstring instanceId;
+    IfFailRet(m_pEnvironment->GetEnvironmentVariable(_T("DotnetMonitorProfiler_InstanceId"), instanceId));
+
+#if TARGET_UNIX
```

Comment:
Do you think we'll add more file system type code in the future? I can see us doing more work in the `/tmp` dir in future chagnes. Maybe we should put this in a helper class so it doesn't need to be repeated.